metals/src/main/scala/scala/meta/internal/pantsbuild/IntelliJ.scala

```diff
@@ -81,6 +84,25 @@ object IntelliJ {
       StandardOpenOption.TRUNCATE_EXISTING,
       StandardOpenOption.CREATE
     )
+
+    val refreshCommand = List(
+      coursier.toString,
+      "launch",
+      s"org.scalameta:metals_2.12:${V.metalsVersion}",
+      "--main",
+      classOf[BloopPants].getName,
+      "--",
+      RefreshCommand.name,
+      "--workspace",
+      project.common.workspace.toString,
+      project.name
+    )
+    val settings = WorkspaceSettings(None, None, Some(refreshCommand))
```

Comment:
We should read the base settings and only override the refresh command field. The settings file is for example updated by Bloop when connecting to Metals (for semanticdb version)